src/tests/ftest/util/agent_utils.py

```diff
@@ -81,9 +183,10 @@ def run_agent(basepath, server_list, client_list=None):
                     nodeset, host_type, directory, user))
 
     # launch the agent
-    with open("../../.build_vars.json") as json_vars:
-        build_vars = json.load(json_vars)
-    daos_agent_bin = os.path.join(build_vars["PREFIX"], "bin", "daos_agent")
+    daos_agent_bin_line = "daos_agent -o " + agent_yaml
+    daos_agent_bin = os.path.join(test.basepath, "install/bin",
+                                  daos_agent_bin_line)
```

Comment:
Same comment above about the join method.  Also it is best to use the `os.path.join()` method to handle all the path separators:
```
    daos_agent_bin = os.path.join(test.prefix, "bin", "daos_agent")
    daos_agent_cmd = " ".join(daos_agent_bin, "-o", agent_yaml))
```